samples/msal-node-samples/auth-code-cli-brokered-app/README.md

```diff
@@ -0,0 +1,47 @@
+# MSAL Node Standalone Sample: Brokered Auth
+
+This sample demonstrates an MSAL Node [public client application](../../../lib/msal-node/docs/initialize-public-client-application.md) that lets users authenticate against **Azure AD** using the device broker.
+
+## Setup
+
+Locate the folder where `package.json` resides in your terminal. Then type:
+
+```console
+    npm install
+```
+
+## Register
+
+1. Navigate to the [Azure portal](https://portal.azure.com) and select the **Azure AD** service.
+1. Select the **App Registrations** blade on the left, then select **New registration**.
+1. In the **Register an application page** that appears, enter your application's registration information:
+   - In the **Name** section, enter a meaningful application name that will be displayed to users of the app, for example `msal-node-webapp`.
+   - Under **Supported account types**, select **Accounts in this organizational directory only**.
+   - In the **Redirect URI (optional)** section, select **Public client/native (mobile & desktop)** in the combo-box and enter the following redirect URI: `http://localhost`.
+1. Select **Register** to create the application.
+1. In the app's registration screen, find and note the **Application (client) ID** and **Directory (Tenant) ID**. You use these values in your app's configuration file(s) later.
+
+Before running the sample, you will need to replace the values in the configuration object:
+
+```javascript
+const config = {
+    auth: {
+        clientId: "ENTER_CLIENT_ID",
+        authority: "https://login.microsoftonline.com/ENTER_TENANT_ID",
```

Comment:
Can they use the organization/consumer/common endpoints here?  Does it have to be tenant id?

Reply:
Yes, they can use any of those but this placeholder is the standard we use across our samples

Reply:
+1 for id -probably shouldn't encourage multitenancy by default (though I use `ENTER_TENANT_INFO` to be pedantic)